src/components/datePicker/DatePickerReducers.ts

```diff
@@ -41,20 +41,22 @@ export const datePickersInitialState: IDatePickerState[] = [];
 
 const addDatePicker = (state: IDatePickerState, action: IReduxAction<IAddDatePickerPayload>): IDatePickerState => {
     const mayBeNull = (d: Date) => action.payload.initiallyUnselected ? null : d;
+    const appliedLowerLimit = (action.payload.dateRangeOnRender || [])[0];
+    const appliedUpperLimit = (action.payload.dateRangeOnRender || [])[1];

```

Comment:
Won't getting any index on an empty array fail?

Reply:
it will result in undefined, which was my intent, I don't think this is wrong for the case at hand

Reply:
that's what I thought, but:
![image](https://user-images.githubusercontent.com/33127602/43204816-e4309e42-8fef-11e8-9979-cae7071f8737.png)

Reply:
![image](https://user-images.githubusercontent.com/9539763/43205848-4e492c7a-8ff2-11e8-84bc-6495bb93ad7b.png)

Reply:
see console at the top right of the image